### ISSUE TITLE
Images shared with the share sheet send the added text as separate message instead of as image caption

### ISSUE THREAD

**Issue by hutattedonmyarm** [TelegramMessenger/Telegram-iOS#512]
### Checklist
- [x] I am reporting an issue in existing functionality that does not work as intended
- [x] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
Images shared with the share sheet send the added text as separate message instead of as image caption

### Expected Behavior
When I share an image using the share sheet and add text using the share sheet, I expect the text to be displayed as the image's caption

### Actual Behavior
When I share an image using the share sheet and add text using the share sheet, the text is sent as a separate message

### Steps to Reproduce
1.  View an image somewhere on iOS
2. Press the share button
3. Select Telegram I the share sheet
4. Add some text
5. Tap "Send"

### Screenshots and Videos 
Remove, if not applicable

### Environment
**Device:** `iPhone`

**iOS version**: `14.4.2`
![Sharesheet](https://user-images.githubusercontent.com/664905/113298789-79ee3400-92fc-11eb-81d9-035b0285c2fc.jpg)
![Actual](https://user-images.githubusercontent.com/664905/113298919-9ee2a700-92fc-11eb-8bcf-e9fd03908156.jpg)
![Expected](https://user-images.githubusercontent.com/664905/113298993-b15ce080-92fc-11eb-8c69-8c75e1806be9.jpg)

**App version:** `7.6.1`

**Comment by VetaLioSTV** [comment]
Yes, this is old one problem. Workaround(actually i'm doing it always), send file AS IS,then manually "edit" sent file(Image/audio/nvm) and add text..
